Stylesheets/Components/badge.css

```diff
@@ -0,0 +1,119 @@
+.avatar-badge-container {
+    width:fit-content;
+    height: fit-content;
+    position: relative;
+}
+
+.avatar-badge-container .avatar-container, 
+.avatar-badge-container img {
+    margin:0;
+}
+
+.badge {
+    border-radius: var(--circle-border);
+    position:absolute;
+    right:0;
+    bottom:0;
+    z-index: 9;
+    box-shadow: var(--badge-shadow)
+}
+
+.squircle + .badge {
+    right: -0.5rem;
+    bottom: -0.5rem;
+}
+
+.squircle.xsmall + .badge {
+    right: -0.2rem;
+    bottom: -0.2rem;
+}
+
```

Comment:
Everything else look good , just try to use px for units less than 1rem

Reply:
okay, I'll do the changes. thankyou.